src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,24 @@
+const ArticleCard = (article) => {
+  let formattedDate = new Date(article.created_at).toLocaleDateString("en-us", {
```

Comment:
I like this date formatting going on here - it might be worth extracting this into a little util function